src/components/FilterOverlay.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div id="dialog">
+    <v-app id="filterOverlayDialog">
+      <v-row
+        align="center"
+        justify="center"
+      >
+        <v-overlay
+          absolute
+          :value="overlay"
+        >
+          <v-card>
+            <v-list>
+              <v-list-item>
+                <v-list-item-content>
+                  <v-list-item-title>Action Needed</v-list-item-title>
+                  The network you are loading is too large and additional
+                  parameters are required to visualize. Consider one of the
+                  following:
+                </v-list-item-content>
+              </v-list-item>
+            </v-list>
+            <v-card>
+              <v-divider />
+              <v-card-text>
+                <div>Choose a random subset of edges</div>
+                <v-slider
+                  v-model="subsetAmount"
+                  :max="max"
+                  :min="min"
+                  step="10"
+                  :ticks="true"
+                  thumb-label
+                >
```

Comment:
This will make the label visible
```suggestion
                  thumb-label
                  thumb-color="primary"
                >
```